src/Uno.UI.FluentTheme/Resources/PriorityDefault/NavigationView_themeresources.xaml

```diff
@@ -1,880 +0,0 @@
-<!-- Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT License. See LICENSE in the project root for license information. -->
-<ResourceDictionary xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml" xmlns:controls="using:Microsoft.UI.Xaml.Controls" xmlns:primitives="using:Microsoft.UI.Xaml.Controls.Primitives" xmlns:media="using:Microsoft.UI.Xaml.Media" xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation">
```

Comment:
Is this file removed to use the "System" version of the style in MUX ?

Reply:
Rather than remove the files it'd be better maintainability-wise to exclude them here: https://github.com/unoplatform/uno/blob/master/src/Uno.UI.FluentTheme/FluentMerge.targets#L12
with a comment indicating the reason why.

Reply:
I think these files were actually leftovers from the implementation before hierarchical `NavigationView`, so I think the removal is ok, as they don't exist in the MUX repo either